lighthaus.py

```diff
@@ -23,69 +22,93 @@ def __init__(
             initial_scroll_speed: float = 0,
             sleep_time: float = 0.01,
             brightness: float = 1.0,
+            fade_in_time: float = 5.0,
+            fade_out_time: float = 20.0,
     ):
         self.writer = writer
-        self.color_1 = initial_color_1
-        self.color_2 = initial_color_2
-        self.scroll_speed = initial_scroll_speed
+        self.scheduled_gradient = Gradient(
+                seconds=get_seconds_now(),
+                color_1=initial_color_1,
+                color_2=initial_color_2,
+                scroll_speed=initial_scroll_speed,
+                brightness=brightness
+            )
+        self.current_gradient = self.scheduled_gradient
+        self.user_gradient = Gradient(
```

Comment:
It would be nice to have a comment on what the different gradients mean. It's not immediately obvious to me. 

Also, maybe we want to have the `self.user_graident = None` when it's not active.